solutions/basic/stack_definition.json

```diff
@@ -383,7 +309,7 @@
         },
         {
           "name": "existing_kms_instance_crn",
-          "value": "ref:../../members/2a - Security Service - Key Management/outputs/kms_instance_crn"
+          "value": "ref:../../members/2 - Security Service - Key Management/outputs/kms_instance_crn"
```

Comment:
I think this should follow the ref scheme we used in other places - i.e. `ref:../2 - Security Service - Key Management/outputs/kms_instance_crn`